pkg/controller/cassandracluster/generator.go

```diff
@@ -750,7 +739,17 @@ func createCassandraContainer(cc *api.CassandraCluster, status *api.CassandraClu
 				},
 			},
 		},
-		Env: createEnvVarForCassandraContainer(cc, status, resources, dcRackName),
```

Comment:
There is no need of all the env vars that were push here before ? (Any ideas why they were pushed before ? Just to be sure that we don't miss anything else like I did for the MAX_HEAP :D )

Reply:
the Cassandra container does not need them